platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -633,7 +633,7 @@
     /**
     * The MdsSchemaEditorCtrl controller is used on the 'Schema Editor' view.
     */
-    controllers.controller('MdsSchemaEditorCtrl', function ($scope, $timeout, $http, Entities, MDSUsers, Permissions, MDSUtils, Locale, ModalFactory, LoadingModal) {
+    controllers.controller('MdsSchemaEditorCtrl', function ($scope, $timeout, $http, Entities, MDSUsers, Permissions, MDSUtils, Locale, ModalFactory, LoadingModal, $rootScope) {
```

Comment:
@MichalKus Please remove `$rootScope` from this controller if it is not being used anywhere inside it